src/views/home/components/EditionCard.vue

```diff
@@ -38,68 +62,60 @@ import { EditionInfo } from '@/models/edition';
 import EditionIcons from '@/components/cues/edition-icons.vue';
 
 @Component({
-    name: 'edition-version-card',
-    components: { 'edition-icons': EditionIcons },
+    name: 'edition-card',
+    components: { EditionIcons },
 })
-export default class EditionVersionCard extends Vue {
+export default class EditionCard extends Vue {
     @Prop() public edition!: EditionInfo;
 
-    // data() becomes data members
-    //
-    // data {
-    //        return {
-    //          element: 5;
-    //      }
-    // }
-    //
-    // private element = 5;
-
-    // Computed becomes getters
-    //   private get lockEdition(): EditionInfo[] {
-    //         return this.$state.editions.items.filter(
-    //             ed => ed.permission.readOnly
-    //         );
-    //     }
     private get thumbnailSource(): string | undefined {
         return this.edition.thumbnail
             ? this.edition.thumbnail.thumbnailUrl
             : undefined;
     }
-    private get lockEdition(): boolean {
-        return this.edition.permission.readOnly;
-    }
-
-    private get shareCount(): number {
-        return this.edition.shares.length - 1; // One is the current user
-    }
-
-    private get shareNames(): string {
-        const names = this.edition.shares.map(share => share.email);
-        return names.join(', ');
-    }
 }
 </script>
 
 <style lang="scss" scoped>
 @import '@/assets/styles/_variables.scss';
-.card:hover,
-.list-item .card:hover {
-    transform: scale(0.95, 0.95);
-    box-shadow: 5px 5px 20px 5px rgba(0, 0, 0, 0),
-        -5px -5px 20px 5px rgba(0, 0, 0, 0.22);
+@import '@/assets/styles/_fonts.scss';
+
+.cart-title {
+    font-weight: $font-weight-3 !important;
+    color: $black;
+    margin-bottom: 1px;
+}
+
+.cart-decoration:hover {
+    text-decoration: none;
 }
-img.card-img-top {
-    display: block;
-    height: 100px;
-    max-height: 100px;
-    object-fit: cover;
+.card-font {
+    font-style: $font-style;
+    font-weight: $font-weight-1;
+    font-size: $font-size-2;
+    font-family: $font-family;
 }
-h5 {
-    cursor: pointer;
+.card-label {
+    color: $grey;
+    margin-bottom: 1px;
 }
-
-.share {
+.card-date {
     color: $black;
-    background-color: white;
 }
-</style>
\ No newline at end of file
+.status-badge {
+    font-family: $font-family;
+    text-align: center;
+    font-size: $font-size-1;
+    width: 68px;
+    height: 29.58px;
+    line-height: 20px;
+}
+.status-badge-Draft {
```

Comment:
Here's a real nitpick: probably don't use a capital letter in the class name here and below.